Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -152,19 +156,29 @@ private void OpenLauncherLogFileLocation()
         {
 #if UNITY_EDITOR
             OpenPopup("Access to Launcher in the editor is not possible");
+#else
+#if UNITY_STANDALONE_OSX
+            string logDirectoryPath = Patcher.Instance.Data.Value.LockFilePath.Replace(
+                Patcher.Instance.AppSecret + Path.DirectorySeparatorChar + ".lock", "");
```

Comment:
Does it work?

As I remember paths on OSX have shortened, not full, app secret (abcdefgh123456 => abc456)

Reply:
you're right, Hash is the first eight characters. Done